src/java.base/share/classes/java/nio/X-Buffer.java.template

```diff
@@ -2239,10 +2239,10 @@ public abstract sealed class $Type$Buffer
      * and limit will be zero.  If rounding is within bounds the following
      * expressions will be true for a new buffer {@code nb} and unit size
      * {@code unitSize}:
-     * <pre>{@code
+     * {@snippet lang=java :
      * nb.alignmentOffset(0, unitSize) == 0
      * nb.alignmentOffset(nb.limit(), unitSize) == 0
```

Comment:
The indentation (within the files) is inconsistent, I think at 4 spaces might be an improvement.

Reply:
Fixed in https://github.com/openjdk/jdk/pull/13870/commits/acd62d5ccadd0667b2b08aae9de531adbfdcc8b1.